### PR TITLE
Get urls without dependencies

### PR DESCRIPTION
**Pull request by Dankni95**
Heya

This seem to work, had to replace some part of string and have no idea why it came out with quotes and a `b` after encrypting.

Can look into refactoring later!

Test if everything works on your end :+1:

### CODE REVIEW

**Comment by Dankni95** [comment]
Is it me or is my code missing this part:

``` 
if 'peliscdn' in json_data[0]['file']:
            r = self.session.get(json_data[0]['file'], headers={'referer': self.entry.embed_url})
            qualitys = re.findall(r'(?<=\d\d\dx)\d+', r.text)
            quality_links = [x for x in r.text.split('\n')]
            quality_links = [x for x in quality_links if not x.startswith('#')]
            qualitys.reverse()
            quality_links.reverse()
            

I fetched right before the pull request...

File: anipy_cli/url_handler.py [hunk 1]
```diff
@@ -113,74 +116,74 @@ def pick_range(self):
...
-        #cipher = AES.new(private_key, AES.MODE_CBC, iv)
-        #print(base64.b64encode(cipher.encrypt(video_id)))
-        #ajax = base64.b64encode(iv + cipher.encrypt(video_id))
+
+        cryptor = AES.new(self.key, self.mode, self.iv)
+        encrypted = cryptor.encrypt(
+            bytearray(self.pad(video_id+"\n"), "utf-8"))
+        ajax = base64.b64encode(encrypted)
+
+        return str(ajax).replace("'", "")[1:]
```
Comment (sdaqo) [inline anipy_cli/url_handler.py:186]: You can use `ajax.decode('utf-8')` instead of `return str(ajax).replace("'", "")[1:]`

**Comment by sdaqo** [comment]
> Is it me or is my code missing this part

Yes it is, weird because this commit was 5 hours ago. Could you add it back?

**Comment by Dankni95** [comment]
Hey, added the changes needed!

Also im wondering what kind of LSP/Formatter you are using? If we all unite formatters the file changes are easier to read!

**Comment by sdaqo** [comment]
Hey, thanks for adding. Um I use the inbuilt neovim LSP, what do you use?

**Comment by Dankni95** [comment]
I have no clue what server default neovim lsp uses, tried to google without finding out. 

Are you able to run `:LspInfo` or `:LspInstalled`? This may be a plugin i have tho!

https://github.com/williamboman/nvim-lsp-installer

I use pylsp, but can easely switch for which ever.

image

**Comment by sdaqo** [comment]
It's pyright, I'm pretty sure

**Pull request merged**